job_runner/main.py

```
@@ -97,17 +192,27 @@ async def send_job_update(job_id: str, update: JobUpdate) -> None:
     """Send update via Redis pub/sub only (no HTTP)"""
     try:
         channel = f"job:{job_id}:updates"
+        update_payload = update.model_dump(mode="json", exclude_none=True)
+        existing_status_raw = await redis_client.get(f"job:{job_id}:status")
+        existing_status = json.loads(existing_status_raw) if existing_status_raw else {}
+        merged_status = {
+            **existing_status,
+            **update_payload,
+            "job_id": existing_status.get("job_id", job_id),
+            "updated_at": datetime.now().isoformat(),
```

Ruff (0.15.10)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)